about.md

```diff
@@ -152,42 +152,90 @@ tax adviser or the IRS about your particular tax situation.
 
 ## Funding
 
-We are grateful for financial support from [NumFOCUS][] and the [Chan
-Zuckerberg Initiative][] (under an [EOSS4
-award](https://chanzuckerberg.com/eoss/proposals/mdanalysis-faster-extensible-molecular-analysis-for-reproducible-science/)).
+We are grateful for financial support from the following organizations, which have supported MDAnalysis either through direct funding or indirectly by funding MDAnalysis contributors.
 
-Earlier work was supported by the [National Science
-Foundation](https://www.nsf.gov/) (as part of award ACI-1443054). 
-
-We have been been participating in [Google Summer of
-Code](https://summerofcode.withgoogle.com/) since 2016 and are
-grateful for Google's support.
-
-
-
-
-<a href="{{site.numfocus.sponsored_project}}"><img
-    src="{{site.images}}/numfocus-sponsored.png" title="NumFOCUS
-    sponsored project" alt="NumFOCUS Sponsored" style="display:
-    inline; float: left; height: 4em; margin: 0 0.5em" /></a>
+### [Chan Zuckerberg Initiative][] (CZI)
+MDAnalysis has been supported by the [Essential Open Source for Science](https://chanzuckerberg.com/rfa/essential-open-source-software-for-science/) (EOSS) program from the CZI DAF, an advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989)
 
+- EOSS5, 2022-253062 (**2022**): [MDAnalysis: Outreach and Project Manager](https://chanzuckerberg.com/eoss/proposals/mdanalysis-outreach-and-project-manager/) (**Key Personnel**: @IAlibay, @micaela-matta, @orbeckst, @richardjgowers (*PI*); **Hired Personnel**: @jennaswa)
+- EOSS4, DAF2021-237663, DOI [https://doi.org/10.37921/426590wiobus](https://doi.org/10.37921/426590wiobus) (**2021**): [MDAnalysis: Faster, Extensible Molecular Analysis for Reproducible Science](https://chanzuckerberg.com/eoss/proposals/mdanalysis-faster-extensible-molecular-analysis-for-reproducible-science/) (**Key Personnel**: @fiona-naughton, @IAlibay, @lilyminium, @orbeckst (*PI*), @richardjgowers; **Hired Personnel**: @ianmkenney)
```

Comment:
@hmacdope is missing somewhere here - not sure if key

Reply:
Added Hugo to EOSS4